packages/webservice/app/src/main/resources/models/tables/queryStats.hjson

```diff
@@ -0,0 +1,113 @@
+{
+    tables:
+    [
+        {
+            name: QueryStats
+            table: QueryStats
+            description:
+            '''
+            Carbon Query Stats Table
+            '''
+            cardinality: large
+            measures :
+            [
+                {
+                    name: durationInMillis
+                    type: DECIMAL
+                    definition: IF (durationInMillis IS NULL, -1, durationInMillis)
+                }
+                {
+                    name: rowsReturned
+                    type: DECIMAL
+                    definition: IF (rowsReturned IS NULL, -1, rowsReturned)
+                }
+                {
+                    name: bytesReturned
+                    type: DECIMAL
+                    definition: IF (bytesReturned IS NULL, -1, bytesReturned)
+                }
+            ]
+            dimensions :
+            [
+                {
+                    name: requestId
+                    type: TEXT
+                    definition: requestId
+                }
+                {
+                    name : apiVersion
+                    type : TEXT
+                    definition : IF (apiVersion IS NULL, 'Unknown', apiVersion)
+                }
+                {
+                    name : apiQuery
+                    type : TEXT
+                    definition : IF (apiQuery IS NULL, 'Unknown', apiQuery)
+                }
+                {
+                    name : storeQuery
+                    type : TEXT
+                    definition : IF (storeQuery IS NULL, 'Unknown', storeQuery)
+                }
+                {
+                    name : isCached
+                    type : BOOLEAN
+                    definition : DEFAULT (false)
+                }
+                {
+                    name : hostName
+                    type : TEXT
+                    definition : IF (hostName IS NULL, 'Unknown', hostName)
+                }
+                {
+                    name : createdOn
+                    type : TIME
+                    definition : createdOn
+                    grains: [
+                    {
+                        grain :  MONTH
```

Comment:
Elide currently defines the following time grains:

    SIMPLEDATE("yyyy-MM-dd"),
    DATETIME("yyyy-MM-dd HH:mm:ss"),
    MONTHYEAR("MMM yyyy"),
    YEARMONTH("yyyy-MM"),
    YEAR("yyyy"),
    WEEKDATEISO("yyyy-MM-dd")

Reply:
Here is the new set of grains: https://github.com/yahoo/elide/pull/1608